app/src/main/java/com/palone/planahead/screens/home/ui/HomeScreen.kt

```diff
@@ -37,7 +37,8 @@ fun HomeScreen(viewModel: HomeScreenViewModel, navHostController: NavHostControl
     val uiState = viewModel.uiState.collectAsState().value
     val sheetState = rememberModalBottomSheetState()
     val uiScope = rememberCoroutineScope()
-    val tasks = viewModel.allTasks.collectAsState(initial = emptyMap())
+    val tasks =
+        viewModel.uiState.collectAsState().value.allTasks.collectAsState(initial = listOf()) // not sure about this one (it looks awful)
```

Comment:
```suggestion
    val tasks = uiState.allTasks.collectAsState(initial = listOf())
```

Reply:
Well, it yells at me ".value shouldn't be called within composition", so it has to stay as it is I guess